packages/core/src/components/popover-core/popover-core.tsx

```diff
@@ -225,15 +232,21 @@ export class TdsPopoverCore {
     this.popperInstance?.destroy();
   }
 
-  componentDidLoad() {
+  connectedCallback() {
+    if ((this.selector === undefined && this.referenceEl === undefined) || this.disableLogic) {
+      this.disableLogicState = true;
+      console.warn('TDS-POPOVER-CORE: Popover internal logic disabled.');
+      return;
+    }
+
     this.initialize({
       referenceEl: this.referenceEl,
       trigger: this.trigger,
     });
   }
 
   componentDidRender() {
-    if (this.isShown && !this.renderedShowValue) {
+    if (this.isShown && !this.renderedShowValue && !this.disableLogicState) {
```

Comment:
@mJarsater  - State is made is other parts of components logic can be disabled.

Reply:
Yeah but I think that rather than adding a prop that disables everything I would take a step back and see if there is a pattern you could follow for this and the same type of components. 

Ideally you'd only have to implement controlled vs uncontrolled in `popover-core` and then just forward the props in other component.